src/modules/auth/index.js

```diff
@@ -33,6 +34,13 @@ export default class AuthModule extends Module {
 			name: 'Access control',
 			component: AccessControlScreen
 		});
+
+		app.Router.addRoute({
+			path: '/auth/about',
```

Comment:
About screen should not be part of the "Auth" module - it is "own" dedicated module.

Reply:
I made some changes, could you please review them?